pkg/controller/services.go

```diff
@@ -932,6 +938,13 @@ func (cont *AciController) opflexDeviceChanged(obj apicapi.ApicObject) {
 			for node, devices := range cont.nodeOpflexDevice {
 				if node == obj.GetAttrStr("hostName") {
 					for _, device := range devices {
+						if obj.GetAttrStr("hostName") == name {
+							if device.GetDn() != obj.GetDn() && device.GetAttr("state") == "connected" {
+								cont.log.Info("Found a device that is in connected state, this happens during a live migration")
```

Comment:
I would rephrase this something to the effect of:

Found two opflex devices associated with this node, one in connected state and another one transitioned to disconnected state.